src/sqlfluff/rules/L040.py

```diff
@@ -15,6 +15,40 @@ class Rule_L040(Rule_L010):
     """Inconsistent capitalisation of boolean/null literal.
 
     The functionality for this rule is inherited from :obj:`Rule_L010`.
+
+    | **Anti-pattern**
+    | In this example, 'null' and 'false' is in lower-case whereas 'TURE' is in upper-case.
+
+    .. code-block:: sql
+
+        select
+            a,
+            null,
+            TURE,
+            false
+        from foo
+
+    | **Best practice**
+    | Make all literal null/true/false either in upper-case or in lower-case
+
+    .. code-block:: sql
+
+        select
+            a,
+            null,
+            true,
+            false
+        from foo
+
+        -- Also good
+
+        select
+            a,
+            NULL,
+            TRUE,
+            FALSE
+        from foo
```

Comment:
Change order to match text:

```suggestion
        select
            a,
            NULL,
            TRUE,
            FALSE
        from foo

        -- Also good

        select
            a,
            null,
            true,
            false
        from foo
```